src/components/Kv/KvSettingsCard.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<div class="settings-card">
+		<div class="icon-wrapper">
+			<slot name="icon"></slot>
+		</div>
+		<div class="title-wrapper">
+			<h3>{{ title }}</h3>
+		</div>
+		<div class="content-wrapper">
+			<slot name="content"></slot>
+		</div>
+	</div>
+</template>
+
+<script>
+
+export default {
+	props: {
+		title: {
+			type: String,
+			default: ''
+		},
+	},
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.settings-card {
+	background: $white;
+	padding: 1.95rem;
+	display: grid;
```

Comment:
Does this component still render semi-ok in IE?

Reply:
@ryan-ludwig It doesnt look too bad without the grid support. 

<img width="990" alt="Screen Shot 2020-08-06 at 4 12 15 PM" src="https://user-images.githubusercontent.com/4371888/89591696-b2864e00-d7ff-11ea-8a18-faabf645da9b.png">

Reply:
Yeah, not bad at all, nice!

Reply:
@ryan-ludwig I decided to refactor this to use flex instead for better IE support. There really was no reason to use grid here.